test/unit/strings_spec.lua

```diff
@@ -140,13 +147,22 @@ end)
 
 describe('vim_snprintf()', function()
   local function a(expected, buf, bsize, fmt, ...)
-    eq(#expected, strings.vim_snprintf(buf, bsize, fmt, ...))
+    local args = { ... }
+    local ctx = string.format('snprintf(buf, %d, "%s"', bsize, fmt)
+    for _, x in ipairs(args) do
+      ctx = ctx .. ', ' .. tostring(x)
+    end
+    ctx = ctx .. string.format(') = %s', expected)
+    eq(#expected, strings.vim_snprintf(buf, bsize, fmt, ...), ctx)
     if bsize > 0 then
       local actual = ffi.string(buf, math.min(#expected + 1, bsize))
       eq(expected:sub(1, bsize - 1) .. '\0', actual)
```

Comment:
Probably also want to provide that `ctx` information for this assertion as well.

Reply:
Yes, I ran into the same issue and was going to add that in a follow-up PR. However, I haven't had time to figure out the fix for the other issues you (and I) ran into.

Any help is appreciated.